packages/assistify-help-request/config.js

```diff
@@ -26,6 +26,14 @@ Meteor.startup(() => {
 		section: 'General'
 	});
 
+	RocketChat.settings.add('Assitify_Deactivate_request_closing_comments', false, {
```

Comment:
I do not want to test my German skill here :). So Please help to translate.

Reply:
"Kommentieren beim Schließen von Anfragen verhindern"